File: vmc/beginning/l 2.10/stats.c
#include<stdio.h>

int main()
{
    float min=100000000;
    float max = 0;
    float sum = 0;
    int number = 0;
    float average = 0;
    while (1<2)
    {

        float x ;
        printf("enter your number : ");
        scanf("%f", &x);
        number = number + 1;
        if(x!=-1)
        {
            if(x > max)
                max = x ;
        }
        if(x!=-1)
        {
            if(x < min)
                min = x ;
        }
        if(x!=-1)
        {
            sum = sum + x ;
            average = (sum / number);
        }

        if( x == -1)
        {
            break;
        }
        printf("\n\nThe maximum = %f\n", max);
        printf("The minimum = %f\n", min);
        printf("The average = %f\n", average);
        printf("The number of entries = %d\n\n", number);
    }
}

File: vm/beginning/l 2.10/stats.py
min = 0
max = 0
sum = 0
number = 0
while True :
    x = int(input("enter your number : "))
    number = number + 1
    if x != -1 :
        if x > max:
            max = x
    if x != -1 :      
        if x < min :
            min = x
    if x != -1 :
        sum = sum + x 
        average = int(sum/number) 
    print("\n\n")
    print("The maximum = ", max)           
    print("The minimum = ", min) 
    print("The average = ", average)
    print("The number of entries = ", number) 
    print("\n\n")
    if x == -1 :
        break
      